_health-care/_js/containers/NameAndGeneralInfo.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+
+import NameAndGeneralInfoSection from '../components/personal-information/NameAndGeneralInfoSection';
+import { updateField } from '../actions';
+
+function mapStateToProps(state) {
+  console.log(state);
+  return {
+    data: state.nameAndGeneralInformation,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    onStateChange: (field, update) => {
+      dispatch(updateField(field, update));
+    }
+  };
+}
+
+const NameAndGeneralInfo = connect(mapStateToProps, mapDispatchToProps)(NameAndGeneralInfoSection);
+
+export default NameAndGeneralInfo;
```

Comment:
Right...I think we need to create a consistent naming convention.

For now, I'd like to leave it as is.

My belief is that *this* is going to become the most common component to use when we refer to NameAndGeneralInfo and that we should rename everything else.